CDP4SiteDirectory/ViewModels/RoleBrowser/ParticipantPermissionRowViewModel.cs

```diff
@@ -65,6 +65,17 @@ public ClassKind ShortName
             }
         }
 
+        /// <summary>
+        /// Gets the Access right to display it in the "AccessType" column of the Role browser.
+        /// </summary>
+        public string AccessType
```

Comment:
The usual way that we do it is instead of introducing a property like this, is we bind to the `this.AccessRight` property and implement rather a Converter that does this `ToString()`. Please do this in this case as well.